docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,778 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #155.            stop_idx = start_idx + m
</pre><p>Would it make more sense to initialize both <code>start</code> and <code>stop</code>   OUTSIDE the for loop and then update both at the very end of the for loop? Would that simplify things?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I changed the design a little bit and initialize them outside of the for-loop. I think it should be more readable now.